src/sweeper/__main__.py

```diff
@@ -20,14 +20,18 @@
 import sys
 from docopt import docopt
 from .sweepers.empties import EmptyTest
-import report
+from . import report, backup
 
 def main():
     '''Main entry point for program. Parse arguments and pass to sweeper modules
     '''
 
     args = docopt(__doc__, version='1.0.0')
 
+    # backup file before quality checks
+    if args['--backup-to']:
+        backup.backup_data(args['--workspace'] + '\\' + args['<table_name>'], args['--backup-to'])
```

Comment:
it is not a good practice to join file paths this way. use `os.path.join` instead.